lib/netsuite/records/employee.rb

```diff
@@ -1,20 +1,23 @@
 module NetSuite
   module Records
-
     class Employee < Support::Base
+      include Support::Fields
```

Comment:
I ran a quick test through my app, and this include doesn't seem to be necessary.  I'll take it out.